acceptor.go

```diff
@@ -100,6 +101,11 @@ func (a *Acceptor) Stop() {
 	a.sessionGroup.Wait()
 }
 
+func (a *Acceptor) RemoteAddr(sessionID SessionID) (net.Addr, bool) {
```

Comment:
Since you are adding to the package api, could you add a comment here describing the method for api users that will be picked up by godoc?

Reply:
Added comment